app/players/elves/elf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
```

Comment:
Do not repeat abstract methods from a parent if you have an abstract class that inherits another abstract class.
Since `Player` is already a child of the `ABC` class, you can inherit just from `Player`.